index.html

```diff
@@ -0,0 +1,362 @@
+                See project
+            </a>
+        </div>
+    </div>
+
+    <!-- End of fourth card -->
+
+    <!-- End works section -->
+
+    <section class="about">
+        <div class="am">
+            <h2>
+                About Myself
+            </h2>
+        </div>
+        <div class="disc">
+            <p>
+                Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+        </div>
+        <div class="lets_contact">
+            <div class="contact">
+              <h2>LET’S CONNECT</h2>
+            </div>
+        </div>
+        <section class="social_media">
+            <ul class="social-icons-set">
+              <li class="social-icon">
+                <a href="#"><i class="fa-brands fa-twitter" target="_blank" aria-label="Twitter"></i></a>
+              </li>
+              <li class="social-icon">
+                <a href="#"><i class="fa-brands fa-linkedin-in" target="_blank" aria-label="Linkedin"></i></a>
+              </li>
+              <li class="social-icon"><a href="#"><i class="fa-solid fa-m" target="_blank"
+                    aria-label="Medium"></i></a>
+              </li>
+              <li class="social-icon">
+                <a href="#"><i class="fa-brands fa-github" target="_blank" aria-label="GitHub"></i></a>
+              </li>
+              <li class="social-icon">
+                <a href="#"><i class="fa-brands fa-angellist" target="_blank" aria-label="Angellist"></i></a>
+              </li>
+            </ul>
+        </section>
+        <div class="link">
+            <a href="#">
+                get my resume
+            </a>
+        </div>
+        <div class="select">
+            <ul class="lang" >
+                <li class="lang">
+                    <div class="lang">
+                        <h2>Language</h2>
+                        <i  class="fa fa-angle-down" aria-hidden="true"></i>
+                    </div>
+                    <div class="contain">
+                        <ul>
+                            <li class="con"><img src="js.png" alt=""> JavaScript</li>
+                            <li class="con"><img src="html.png" alt=""> Html</li>
+                            <li class="con"><img src="css.png" alt="">Css</li>
+                        </ul>
+                    </div>
+                </li>
+                <li class="lang">
+                    <div class="framework">
+                        <h2>Fremeworks</h2>
+                        <i  class="fa fa-angle-right" aria-hidden="true"></i>
+                    </div>
+                </li>
+                <li class="lang">
+                    <div class="skill">
+                        <h2>Skills</h2>
+                        <i  class="fa fa-angle-right" aria-hidden="true"></i>
+                    </div>
```

Comment:
- [ ] (about) 3. Kindly increase the top and bottom padding of the "JavaScript", "HTML", and CSS containers, and also make it stretch out to the right a bit, and like correct the "Html" to "HTML" in uppercase likewise "Css" to "CSS" to following the correct spelling guidelines for professionalism.